### DISCUSSION TITLE
How to develop for truffle?

### DISCUSSION THREAD

**Discussion by LuisUrrutia** [ConsenSys-archive/truffle#5585] [General Q&A]
I want to contribute to this repo, and I already think I have a fix for an open issue, but I don't know how to run a command using the cloned truffle.

**Comment by lsqproduction** [comment]
Hello, @LuisUrrutia.  👋 
Thank you for contributing. 
You may find this repo useful while developing for Truffle. https://github.com/trufflesuite/dotfiles
Feel free to comment on the open issue as well when questions arise and let us know which issue you're taking. 
Cheers.
